### ISSUE TITLE
Render field of type picture in PDF does not work properly

### ISSUE THREAD

**Issue by dasistwas** [dasistwas/moodle-mod_datalynx#42]
When I have an upload field of type picture and I upload multiple picture and then integrate the pictures in PDF view, not all pictures are visible in the PDF. A save way to render would be to include one picture per page. Not sure how to assure proper integration of images in the PDF otherwise.

**Comment by dasistwas** [comment]
The attached picture is not shown. Not sure what is different compared to others of same type (png)
![Screenshot_2019-05-15 Jahrbuch Übersicht Seite 1(1)](https://user-images.githubusercontent.com/1620975/58387300-cc79ad80-800c-11e9-86ef-70fedd31c683.png)

**Comment by dasistwas** [comment]
Issue is only with png files and not with every png file. Quite weird bug.

**Comment by michaelpollak** [comment]
If I set up a new instance with a field picture and a view pdf,
I can add an entry with the png you have sent and a second png,
save it and "Export all" in a pdf. 
I see them as expected below each other. Tested also with jpgs as well as your png and a random jpg. 

Possibly uploading to github distorts the image and "fixes" the issue with this specific file?
When opening your file in GIMP I see an embedded color profile, but this should be fine as well. Sorry but at the moment I can't reproduce this report.
